pkgs/applications/version-management/git-and-tools/git-publish/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, python, perl, fetchFromGitHub }:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
{ lib, stdenv, python, perl, fetchFromGitHub }:

stdenv.mkDerivation rec {
```

Reply:
Do we have a written convention for this now? If not I won't apply this.

Reply:
@teto How about you are not being passive aggressive?

Reply:
@lheckemann we do not have written rules for literally everything and I don't want to write rules for everything. It would make it a lot easier if you could just incorporate style changes together with the other 3 changes. If it would be the only comment then I would have completely ignored it. Thank you.

Reply:
The thing is: I prefer the style that I wrote. As long as we don't have a rule saying what the preferred variant is to resolve matters of pure opinion like this, I don't think it makes sense to have these discussions.

I'm grateful for your reviews that bring my attention to useful things like `installManPage` -- I think we can all agree that using abstractions like this where they exist makes sense.

But things that are matters of pure opinion, like this blank line here, are a waste of time. I'd appreciate if you could limit yourself to the actually useful things -- or policies that are actually written somewhere, so i don't have to guess what "SuperSandro2000-conforming style" is -- in future reviews of my code.